src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,44 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int MAX_CAPACITY = 10;
+    private K[] arrayKey;
+    private V[] arrayValue;
+    private int size;
+
+    public StorageImpl() {
+        this.arrayKey = (K[]) new Object[MAX_CAPACITY];
+        this.arrayValue = (V[]) new Object[MAX_CAPACITY];
```

Comment:
we can initialize fields without the word "this".
Example from common mistakes:
![image](https://user-images.githubusercontent.com/36710718/151787140-742bc1bd-df98-433c-ac3f-a562b3a12038.png)
```suggestion
        arrayKey = (K[]) new Object[MAX_CAPACITY];
        arrayValue = (V[]) new Object[MAX_CAPACITY];
```